### PR TITLE
Added contact us page

### LINKED ISSUE #51: Add contact us form in the contact section.
I want to make the contact us section better by adding a form. Please assign me this issue.

### PR DESCRIPTION
**Pull request by harshb02**
# Description
Added the contact us page

Please include a summary of the change and which issue is fixed.
Fixes #51 

## List any dependencies that are required for this change 
NA

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
On my local host

## UI /UX changes
   Attach gif or screenshot for changes.
  
screencapture-file-C-Users-HARSH-Desktop-Algo-Phantoms-Frontend-contact-html-2021-04-30-13_14_47

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by ravikr126** [comment]
please remove conflicts then i will merge this pr @harshb02

**Comment by harshb02** [comment]
> please remove conflicts then i will merge this pr @harshb02

On it

**Review by ravikr126 — CHANGES_REQUESTED** [review]
remove the contact us section on the main page and change the navbar link to this page.

**Comment by harshb02** [comment]
> remove the contact us section on the main page and change the navbar link to this page.

Done with the changes you have requested and have resolved the conflicts also @ravikr126

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.10.1/devicon.min.css"> 
+    <link rel="apple-touch-icon" sizes="180x180" href="favicon_package\apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="favicon_package\favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="favicon_package\favicon-16x16.png">
+    <link rel="manifest" href="favicon_package\site.webmanifest">
+    <link rel="mask-icon" href="favicon_package\safari-pinned-tab.svg" color="#3caea3">
+    <meta name="msapplication-TileColor" content="#2b5797">
+    <meta name="theme-color" content="#ffffff">
+
+    <title>Algo Phantoms</title>
```
Comment (harikesh409) [inline contact.html:37]: Change the title to Algo Phantoms | Contact

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+                            <a class="dropdown-item" href="DSA.html">RoadMap to your DSA</a>
+
+                            <a class="dropdown-item" href="https://github.com/Algo-Phantoms" target="_blank">Contribute
+                                to Project</a>
+
+                            <a class="dropdown-item" href="practicedsa.html">DSA Problems</a>
+                             <a class="dropdown-item" href="booklist.html">Suggested Books</a>
+                        </div>
+                        <li class="nav-item">
+                        <a class="nav-link  font-weight-bold nav-items" href="contact.html">Contact</a>
```
Comment (harikesh409) [inline contact.html:81]: Make the link as # as it is the current page.

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+            <a class="navbar-brand" href="#"><img src="images/icon.png" alt="Algo Phantoms" width="50" height="50"><strong><span class="brand1">Algo</span>Phantoms</strong></a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+                aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link font-weight-bold nav-items" href="#">Home <span
```
Comment (harikesh409) [inline contact.html:52]: Make the link point to index.html

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+                                class="sr-only">(current)</span></a>
+                                <hr class="hr">
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link font-weight-bold nav-items" href="#about">About</a>
```
Comment (harikesh409) [inline contact.html:57]: Make the link point to index.html#about

File: contact.html [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+                            <label for="subject">Subject <span>*</span></label>
+                            <input type="text" class="form-control" id="subject" placeholder="Enter your Subject ">
+                        </div>
+                        <div class="form-group">
+                            <label for="message">Comment or Message <span>*</span></label>
+                            <textarea class="form-control" id="message" rows="3"></textarea>
+                        </div>
+                        <div class="head_button text-center">
+                            <button class="btn btn-common_head4" id="submit" type="submit">Submit</button>
+                            <div id="msgSubmit" class="h3 text-center hidden"></div>
```
Comment (harikesh409) [inline contact.html:140]: What is the use of this div?

File: contact.html [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+                    <div class="">
+                        <img src="./images/icon.png" alt="" class="img-fluid mb-lg-4" width="100" height="100">
+
+                    </div>
+                    <p class="text-white">An Open Source organization focuses on Data Structure and Algorithms.</p>
+                </div>
+                <div class="col-lg-2 col-md-2 col-sm-6 col-12 offset-lg-1">
+                    <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Useful Links</h5>
+                    <ul>
+                        <li>
```
Comment (harikesh409) [inline contact.html:164]: Point it to index.html#about.

File: css/style.css [hunk 7]
```diff
@@ -63,10 +63,6 @@ iframe {
 
 /******************* HEADER ******************/
 
-.top{
```
Comment (harikesh409) [inline css/style.css:66]: Why did you remove this?

File: css/style.css [hunk 8]
```diff
@@ -730,62 +812,6 @@ HEAD .round-2 {
 
 /*
 ...........responsive.............*/
-@media only screen and (max-width: 600px) {
```
Comment (harikesh409) [inline css/style.css:733]: Why is this removed?

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by harshb02** [comment]
Resolved the changes @harikesh409

**Comment by harikesh409** [comment]
> remove the contact us section on the main page and change the navbar link to this page.

This is not done.

**Comment by harshb02** [comment]
> > remove the contact us section on the main page and change the navbar link to this page.
> 
> This is not done.

now it is done @harikesh409

**Review by harikesh409 — APPROVED** [review]

**Review by ravikr126 — APPROVED** [review]

**Pull request merged**